src/schema/lessonContent.schema.ts

```diff
@@ -89,6 +89,13 @@ export const lessonContentSchema = z.object({
   supplementary_asset_object_url: z.string().nullable(),
   worksheet_asset_object_url: z.string().nullable(),
   slide_deck_asset_object_url: z.string().nullable(),
+  lesson_guide_asset_id: z.number().nullable().optional(),
```

Comment:
Can you add comments to these to instruct that the optional flag should be removed once lesson_content_3 is retired ?

Reply:
Comment added 🫡